group-3/homeworks/Homework2 AJS/script.js

```diff
@@ -0,0 +1,79 @@
+let calculate = function(array, operator) {
+    let sum = 0;
+
+    for (var i = 0; i < array.length; i++) {
+
+        let oneCalc=0;
+
+        switch (operator) {
```

Comment:
Right now the switch will run for every element of the array - and the operation stays the same all the time. Try to use a functional approach to make sure that the switch only runs once per array